pkg/karmadactl/interpret/check.go

```diff
@@ -13,6 +13,10 @@ import (
 	"github.com/karmada-io/karmada/pkg/resourceinterpreter/configurableinterpreter/luavm"
 )
 
+var (
+	checkErr = cmdutil.ErrExit
```

Comment:
It doesn't smell good.
I know you want to test the whole command, not just the `runCheck` method.
That brings complexity to both the code and tests.

Just post my feeling, and leave it to @lonelyCZ and @XiShanYongYe-Chang to decide.

Reply:
I think so, we should perform unit tests at the smallest possible granularity.